### PR TITLE
test(alert): update demo html to match Figma

### PR DESCRIPTION
**Pull request by ManahariDahal**
**Related Issue:** #2436

## Summary
- match the current calcite-alert demo page according to the Figma design 
- "_autodismiss speed_" DOM manipulation should work as intended

### CODE REVIEW

**Comment by bstifle** [comment]
image

1. let's drop these labels down to avenir-medium for the weight


image
image

2. let's make the audodismiss speed selector a radio-group with the selection clear appearance


image

3. the "using the serFocus()"  doesn't seem to work

**Comment by ManahariDahal** [comment]
I realized there were some recent changes with the addition of the sizes _small_, _medium_, and _large_. 
Screen Shot 2021-07-08 at 11 16 56 AM

However, the Figma design already encompasses them. Let me know if I should keep them. The current design is the one on the video attached above.

File: src/demos/calcite-alert.html [hunk 1]
```diff
@@ -7,237 +7,1242 @@
     <title>Calcite Alert</title>
     <script src="_assets/head.js"></script>
     <script src="_assets/createAlert.js"></script>
+
+    <!--
+                      _
+                  .__(.)< (MEOW)
```
Comment (julio8a) [inline src/demos/calcite-alert.html:13]: Remove the MEOW comment

**Review by julio8a — CHANGES_REQUESTED** [review]
This is great! left a comment to remove the meow comment. Also, not sure if we need these:
- Alerts inserted into the DOM programmatically
- Using the setFocus() method (once opened)

I think we can remove them, what do you think @caripizza?

**Comment by caripizza** [comment]
@julio8a I personally like having those examples. It gives us the ability to manually test the behavior in multiple browsers.

**Comment by julio8a** [comment]
Thanks, @caripizza! let's leave them . :)

**Review by bstifle — APPROVED** [review]
lookin good manahari!

File: src/demos/calcite-alert.html [hunk 2]
```diff
@@ -7,237 +7,1237 @@
     <title>Calcite Alert</title>
     <script src="_assets/head.js"></script>
     <script src="_assets/createAlert.js"></script>
+
+    <!-- Internal style -->
```
Comment (caripizza) [inline src/demos/calcite-alert.html:11]: Can we remove the notes like this?

File: src/demos/calcite-alert.html [hunk 3]
```diff
@@ -7,237 +7,1237 @@
...
+            color="red"
+            scale="s"
+            icon-start="x-circle"
+            onclick="closeAlerts()"
+          >
+            Close and remove all alerts
+          </calcite-button>
+
+          <script>
+            let alert = document.getElementsByTagName("calcite-alert");
```
Comment (caripizza) [inline src/demos/calcite-alert.html:721]: use `const`

File: src/demos/calcite-alert.html [hunk 4]
```diff
@@ -7,237 +7,1237 @@
...
+      >
+        <div slot="title">Hello there!</div>
+        <div slot="message">Do you really want to proceed?</div>
+        <calcite-link slot="link" title="my action"> Are you sure? </calcite-link>
+      </calcite-alert>
+    </div>
+
+    <!-- 
+        ******************************************************
+        * MEDIUM PROP
```
Comment (caripizza) [inline src/demos/calcite-alert.html:955]: Let's use the prop name `scale` ("MEDIUM SCALE" or similar)

**Review by ManahariDahal — COMMENTED** [review]
fixed all of the requested changes

**Review by julio8a — APPROVED** [review]
LGTM

File: src/demos/calcite-alert.html [hunk 5]
```diff
@@ -7,237 +7,1217 @@
...
+          </script>
+        </div>
+
+        <div class="second-container-margin">
+          <!-- updates the speed of the autodismiss progress bar i.e. fast, medium, slow -->
+          <p>Autodismiss speed</p>
+          <calcite-radio-group appearance="outline" width="full">
+            <calcite-radio-group-item value="Fast" onclick="autoDismissSpeed('fast');"></calcite-radio-group-item>
+            <calcite-radio-group-item
+              value="Moderate"
```
Comment (caripizza) [inline src/demos/calcite-alert.html:737]: Let's match the name of the prop value, "Medium"

File: src/demos/calcite-alert.html [hunk 6]
```diff
@@ -7,237 +7,1217 @@
...
+              alertRemoval.forEach((alert) => {
+                alert.removeAttribute("active");
+              });
+            };
+          </script>
+        </div>
+
+        <div class="second-container-margin">
+          <!-- updates the speed of the autodismiss progress bar i.e. fast, medium, slow -->
+          <p>Autodismiss speed</p>
```
Comment (caripizza) [inline src/demos/calcite-alert.html:733]: Can we match verbiage with the prop? "Auto-dismiss"

**Comment by caripizza** [comment]
@ManahariDahal looks great! Can you add a slotted link or close button inside one of the auto-dismiss examples? It'll be nice to test with the "Focus first item" button too.

**Comment by julio8a** [comment]
This one ready to merge?

**Comment by ManahariDahal** [comment]
> This one ready to merge?

yep, everything seems to work fine

**Pull request merged**